tests/base_test_class.py

```diff
@@ -1,12 +1,14 @@
+import os
```

Comment:
There is no change in this file apart from the order of imports. Makes reviewing the PR unnecessarily more work.

Reply:
That was an IDE thing - I thought I reverted those. Will fix.